dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -81,8 +80,8 @@ public GitHubSourceCodeRepo(String gitUsername, String githubTokenContent, Strin
     public String readFile(String fileName, String reference) {
         checkNotNull(fileName, "The fileName given is null.");
         try {
-            Repository repo = service.getRepository(gitUsername,
-                    gitRepository); // may need to pass owner from git url, as this may differ from the git username
+            // may need to pass owner from git url, as this may differ from the git username
```

Comment:
I think this may just be the case of a poorly named variable, it should already use the valid owner.